.github/workflows/run-test-cases.yml

```diff
@@ -229,6 +220,26 @@ jobs:
       name: Use current version for push
       run: cat version.txt > /tmp/version_to_test.txt
 
+    # For pull_request and pull_request_target, use the files in deployment/helm
+    # as basis for helm install ... this enables us to test any changes made to the
+    # helm chart files in a PR (where no helm chart is published)
+    - if: startsWith(github.event_name, 'pull_request')
+      name: Tell Helm to use the files in deployment/helm to build chart
+      run: |
+        echo './deployment/helm' > /tmp/helm_chart_location.txt
+    # For push, use a specific version of the `akri` charts that are built and
+    # published by the helm workflow.
+    - if: (!(startsWith(github.event_name, 'pull_request')))
```

Comment:
The condition of for the latter two is same, how does that work in practice then?

Reply:
it doesn't.  i was really eager to see if the paths would work using `./deployment/helm` and didn't look carefully.  this needs cleaning up.